test/functional/tests/memory/test_check_number_of_processes.py

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from api.cas import casadm
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.os_utils import (get_number_of_processors_from_cpuinfo,
+                                 get_number_of_processes)
+from test_utils.size import Size, Unit
+
+cleaning_threads_expected = 1
+fill_threads_expected = 0
+metadata_updater_threads_expected = 1
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_check_number_of_processes():
+    """
+        title: Check the number of processes created by CAS.
+        description: |
+          For created CAS device check number of IO threads, cleaning threads, fill threads,
+          metadata updater threads.
```

Comment:
There's also management thread - 1 process for each cache.
name pattern: "cas_mngt_%s"